webrender/src/tiling.rs

```diff
@@ -2190,9 +2196,15 @@ impl Default for PackedStackingContext {
 }
 
 #[derive(Debug, Copy, Clone)]
+enum SimpleCompositeInfo {
+    Opacity(f32),
+    Brightness(f32),
+}
+
+#[derive(Debug, Copy, Clone)]
 enum CompositeKind {
```

Comment:
Add a small comment here explaining what the difference is between simple and complex.